src/Meshes.jl

```diff
@@ -84,6 +84,10 @@ export
   Connectivity,
   facetype, connect, materialize,
 
+  # ordering conventions
+  Ordering, connectivity,
```

Comment:
You are exchanging two terms in this PR: ordering and convention. Can we agree on a single term "OrderingConvention" and use it consistently in the PR?